tox.ini

```diff
@@ -42,11 +42,22 @@ commands =
 
 [testenv:integrationtests]
 basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY BRANCH INTEGRATION_TESTS_REPO_URL
 deps =
 	-rrequirements/tests.txt
+setenv =
+	INT_TESTS_REPO_SSH_URL = git@github.com:gridsingularity/gsy-backend-integration-tests.git
+whitelist_externals =
+	git
+	bash
+commands_pre =
+	pip install -e .
+	git clone {env:INTEGRATION_TESTS_REPO_URL:{env:INT_TESTS_REPO_SSH_URL:}} {envtmpdir}/gsy-backend-integration-tests
 commands =
-    pip install -e .
-	behave --tags=-disabled integration_tests {posargs}
+	bash -c 'cp -r {envtmpdir}/gsy-backend-integration-tests/gsy_e_tests/integration_tests/ ./integration_tests/'
+	bash -c 'for f in ./integration_tests/*.feature; do behave --tags=-slow --tags=-disabled "$f" || exit 1; done'
+commands_post =
+	bash -c '[ -z "{env:INTEGRATION_TESTS_REPO_URL:}" ] && rm -rf ./integration_tests/ || :'
```

Comment:
What happens if we just leave the `integration_tests` folder inside of `{envtmpdir}`? Wouldn't it be automatically deleted at the end of the tests?

Reply:
Update: I now saw your other comment [here](https://github.com/gridsingularity/gsy-e/pull/1367/files#r795839825). I suppose it answers my question.